app/Http/Controllers/CartController.php

```diff
@@ -34,7 +37,7 @@ public function index(Request $request)
 		}
 		$cartitems=CartItem::where('cart_id', $cart->id)->where('product_id', $request->product_id)->first();
 		if($cartitems){
-			$cartitems->update(['quantity' => $cartitems->quantity+1]);
+			$cartitems->update(['quantity' => $cartitems->quantity+1,'price' => $request->price,'discount' => $request->discount,'discount_price' => $request->discount_price]);
```

Comment:
@kams2391 why we are using quantity+1 here?

Reply:
If product item already added in cart and user again add same product then it update that cart item quantity only instead to create new one.

Reply:
ok